### ISSUE TITLE
Some questions about the code

### ISSUE THREAD

**Issue by xiaolonghao** [liuzechun/Nonuniform-to-Uniform-Quantization#1]
eq1:What is the role of the "LearnableBias" class in the code? 
eq2:How to understand the correspondence between  w^r^' in the paper and line 100  in 'resnet.py' in the code. That is “scaling_factor = gamma * torch.mean(torch.mean(torch.mean(abs(real_weights),dim=3,keepdim=True),dim=2,keepdim=True),dim=1,keepdim=True)”
thank you.

**Comment by WANGSSSSSSS** [comment]
bro, no offense, just for the question 2, you d better learn the basics behind the quantization....
